packages/sitecore-jss-nextjs/src/components/RichText.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {useEffect, useRef} from 'react';
+import { useRouter } from 'next/router';
+import { RichText as ReactRichText } from '@sitecore-jss/sitecore-jss-react';
+
+export interface RichTextProps {
+	/** The rich text field data. */
```

Comment:
Not a huge deal, but looks like formatting may be off using tabs instead of spaces (will be nice when we get tslint > eslint conversion done).